2024-05-17 23:50:50,553 INFO    MainThread:3601439 [wandb_setup.py:_flush():76] Current SDK version is 0.16.6
2024-05-17 23:50:50,553 INFO    MainThread:3601439 [wandb_setup.py:_flush():76] Configure stats pid to 3601439
2024-05-17 23:50:50,553 INFO    MainThread:3601439 [wandb_setup.py:_flush():76] Loading settings from /raid/ganesh/racha_suraj/.config/wandb/settings
2024-05-17 23:50:50,553 INFO    MainThread:3601439 [wandb_setup.py:_flush():76] Loading settings from /raid/ganesh/racha_suraj/SF_test/Power-Mechanism-new/Higgs/wandb/settings
2024-05-17 23:50:50,553 INFO    MainThread:3601439 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-05-17 23:50:50,553 INFO    MainThread:3601439 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-05-17 23:50:50,553 INFO    MainThread:3601439 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program_relpath': 'Higgs/priv_higgs_base.py', 'program_abspath': '/raid/ganesh/racha_suraj/SF_test/Power-Mechanism-new/Higgs/priv_higgs_base.py', 'program': '/raid/ganesh/racha_suraj/SF_test/Power-Mechanism-new/Higgs/priv_higgs_base.py'}
2024-05-17 23:50:50,553 INFO    MainThread:3601439 [wandb_setup.py:_flush():76] Applying login settings: {}
2024-05-17 23:50:50,553 INFO    MainThread:3601439 [wandb_init.py:_log_setup():521] Logging user logs to /raid/ganesh/racha_suraj/SF_test/Power-Mechanism-new/Higgs/wandb/run-20240517_235050-8cbtw7zk/logs/debug.log
2024-05-17 23:50:50,553 INFO    MainThread:3601439 [wandb_init.py:_log_setup():522] Logging internal logs to /raid/ganesh/racha_suraj/SF_test/Power-Mechanism-new/Higgs/wandb/run-20240517_235050-8cbtw7zk/logs/debug-internal.log
2024-05-17 23:50:50,554 INFO    MainThread:3601439 [wandb_init.py:init():561] calling init triggers
2024-05-17 23:50:50,554 INFO    MainThread:3601439 [wandb_init.py:init():568] wandb.init called with sweep_config: {}
config: {}
2024-05-17 23:50:50,554 INFO    MainThread:3601439 [wandb_init.py:init():611] starting backend
2024-05-17 23:50:50,554 INFO    MainThread:3601439 [wandb_init.py:init():615] setting up manager
2024-05-17 23:50:50,554 INFO    MainThread:3601439 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2024-05-17 23:50:50,555 INFO    MainThread:3601439 [wandb_init.py:init():623] backend started and connected
2024-05-17 23:50:50,557 INFO    MainThread:3601439 [wandb_init.py:init():715] updated telemetry
2024-05-17 23:50:50,560 INFO    MainThread:3601439 [wandb_init.py:init():748] communicating run to backend with 90.0 second timeout
2024-05-17 23:50:51,205 INFO    MainThread:3601439 [wandb_run.py:_on_init():2357] communicating current version
2024-05-17 23:50:51,248 INFO    MainThread:3601439 [wandb_run.py:_on_init():2366] got version response upgrade_message: "wandb version 0.17.0 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2024-05-17 23:50:51,248 INFO    MainThread:3601439 [wandb_init.py:init():799] starting run threads in backend
2024-05-17 23:50:54,262 INFO    MainThread:3601439 [wandb_run.py:_console_start():2335] atexit reg
2024-05-17 23:50:54,262 INFO    MainThread:3601439 [wandb_run.py:_redirect():2190] redirect: wrap_raw
2024-05-17 23:50:54,262 INFO    MainThread:3601439 [wandb_run.py:_redirect():2255] Wrapping output streams.
2024-05-17 23:50:54,262 INFO    MainThread:3601439 [wandb_run.py:_redirect():2280] Redirects installed.
2024-05-17 23:50:54,263 INFO    MainThread:3601439 [wandb_init.py:init():842] run started, returning control to user process
2024-05-17 23:50:54,263 INFO    MainThread:3601439 [wandb_run.py:_config_callback():1347] config_cb None None {'device': 'cuda:3', 'epochs': 1000, 'lr': 0.0003, 'batch_size': 4096, 'lambda_loss': 1, 'only_reg_flag': 0, 'max_steps': 11000, 'net_depth': 1, 'eps': 0.5}
2024-05-17 23:56:04,501 INFO    MainThread:3601439 [wandb_run.py:_config_callback():1347] config_cb None None {'device': 'cuda:3', 'epochs': 1000, 'lr': 0.0003, 'batch_size': 4096, 'lambda_loss': 1, 'only_reg_flag': 0, 'max_steps': 11000, 'net_depth': 1, 'eps': 0.5, 'grad_norm': 10}
2024-05-17 23:56:20,747 WARNING MsgRouterThr:3601439 [router.py:message_loop():77] message_loop has been closed
